CHANGELOG.md

```diff
@@ -10,6 +10,12 @@
 > - :house: [Internal]
 > - :nail_care: [Polish]
 
+# 10.1.0-rc.6
+
+#### :bug: Bug Fix
+
+- Fix issue where optional field in the inlined record caused the type error https://github.com/rescript-lang/rescript-compiler/pull/5827
```

Comment:
Fix issue where optional fields in inline records were not support and would cause type errors.

Reply:
Thanks https://github.com/rescript-lang/rescript-compiler/pull/5827/commits/bbf2397b0135922b67cae6351ae21b7cb24374c0 👍